Resources/Prototypes/Entities/Objects/Consumable/trash_drinks.yml

```diff
@@ -23,7 +23,33 @@
     interfaces:
     - key: enum.TransferAmountUiKey.Key
       type: TransferAmountBoundUserInterface
-
+  - type: DamageOnLand
+    damage:
+      types:
+        Blunt: 5
+  - type: DamageOtherOnHit
+    damage:
+      types:
+        Blunt: 10
```

Comment:
this damage is too high for other on hit

Reply:
this is the same DamageOtherOnHit for all bottles under drink_bottles.yml maybe change that too?

Reply:
yes